bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceDialog.java

```diff
@@ -194,6 +193,8 @@ public void modifyText(ModifyEvent e) {
 	 * @since 3.0
 	 */
 	private boolean fGiveFocusToFindField = true;
+	private ControlDecoration decoration;
+	private SearchDecoration dec = new SearchDecoration();
```

Comment:
Rename accordingly to the current "naming laws" of this file: add `f`. Also, use descriptive ("full") names for the fields and avoid the "hungarian notation" when possible:

```suggestion
	private ControlDecoration fFindFieldDecoration;
	private SearchDecoration fFindFieldSearchDecoration = new SearchDecoration();
```

Reply:
done, thanks for the naming tips!